manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -618,6 +618,43 @@ data:
     #
     # openshift_csi_share_mount_requests_total measures total amount of attempts to mount csi shared resources into the pods.
     - '{__name__="openshift_csi_share_mount_requests_total"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_logging_info  gives cluster-logging-operator version, managedStatus, healthStatus specific info.
+    - '{__name__="log_logging_info"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_collector_error_count_total  gives cluster-logging-operator deployed collector's (e.g. default collector fluentd) total number of failures in standing it up part of logging pipeline.
+    - '{__name__="log_collector_error_count_total"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_forwarder_pipeline_info  gives cluster-logging-operator deployed logging pipelines info - healthStatus and pipelineInfo as no of total logging pipelines deployed.
+    - '{__name__="log_forwarder_pipeline_info"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_forwarder_input_info  gives cluster-logging-operator logging pipelines input types info - namely application, infra, audit type of log sources input.
+    - '{__name__="log_forwarder_input_info"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_forwarder_output_info  gives cluster-logging-operator logging pipelines output types info - namely to which output end point logs are being directed for further pushing them to a persistent storage.
+    - '{__name__="log_forwarder_output_info"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_collected_bytes_total  gives total bytes collected by the collector, aggregated at each cluster level
+    - '{__name__="cluster:log_collected_bytes_total:sum"}'
+    #
+    # owners: (@openshift/core-loging)
+    #
+    # log_logged_bytes_total  gives total bytes collected by the collector, aggregated at each cluster level
+    - '{__name__="cluster:log_logged_bytes_total:sum"}'
```

Comment:
Does this mean we're still forwarding the raw metric (instead of the recorded `node:log_collected_bytes_total:sum`)?

Reply:
@moadz cluster:log_logged_bytes_total:sum is a name of recording rule that we have set it in the assets/cluster-monitoring-operator/prometheus-rule.yaml to aggregate raw metric log_logged_bytes_total and log_collected_bytes_total.  I see that i mistakenly given record name as node:.. will correct it. thanks for your review.